src/renderer/Renderer.ts

```diff
@@ -301,8 +300,13 @@ export class Renderer {
                 vec4.transformMat4(vector, vector, this.camera.getTransform());
 
                 // Scale them and place them in the lower left corner of the screen
-                vec4.scale(vector, vector, Math.min(this.width, this.height) * 0.05);
-                vec4.add(vector, vector, [this.width * -0.25, this.height * -0.25, -500, 1]);
+                vec4.scale(vector, vector, 20);
+                vec4.add(vector, vector, [
+                    this.width * -0.3,
+                    this.height * -0.3,
+                    -Math.min(this.width, this.height),
```

Comment:
aight so I tried to think of a meaningful comment to put here. Unfortunately I don't really have one. The issue was that as the screen gets smaller, the crosshairs move closer to the middle of the frame:

![screen shot 2018-06-19 at 2 37 51 pm](https://user-images.githubusercontent.com/5315059/41617408-66eebeba-73ce-11e8-8536-3d5e8f2b1095.png)

My gut told me that I should be moving the crosshairs back in the z axis an amount that's relative to the screen size instead of a fixed amount. So I tried this and it worked.  ¯\\\_(ツ)\_/¯